gammapy/astro/population/spatial.py

```diff
@@ -8,7 +8,8 @@
 
     Useful for automatic processing.
 """
-from __future__ import absolute_import, division, print_function, unicode_literals
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
```

Comment:
Please leave this as single line. I know it's different in Astropy, but we have it as 1 line 100x in Gammapy.